### PR TITLE
Add python_requires to setup.py

### PR DESCRIPTION
**Pull request by rouge8**
This will prevent pip from installing this on Python 2.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
     author="Pavel Dedík",
     author_email="pavel.dedik@kiwi.com",
     packages=find_packages(),
+    python_requires=">=3",
```
Comment (goodspark) [inline setup.py]: 3.5 if that matters/possible. What breaks Python 2 are type annotations, which were added in 3.5

File: setup.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
     author="Pavel Dedík",
     author_email="pavel.dedik@kiwi.com",
     packages=find_packages(),
+    python_requires=">='3.5'",
```
Comment (paveldedik) [inline setup.py]: Shouldn't this be -
```suggestion
    python_requires=">=3.5",
```

I'm not sure it will work otherwise, just to be sure.

Comment (rouge8) [reply setup.py]: Fixed!

Comment (paveldedik) [reply setup.py]: That was fast, thx!

**Pull request merged**